ODP-Runtime.md

```diff
@@ -124,61 +121,54 @@ Compliance
 
 In order to locate common resources, some commonly set configuration details are necessary. 
 
--   All environment variables discussed in this section MUST be set on all nodes.
-> Comment (SCG): We need to discuss the scope of these variables. Are they automatically available to every user that logs into the machine, or is there a standard location in which an "-env.sh" script can be found to set the environment variables?
-
--   An ODP Platform MUST set the JAVA\_HOME.
-> Comment (SCG): We had discussed this being available via the "hadoop" script, is this requiring it to be set globally? 
+-   An ODP Platform MUST either explicitly set `JAVA_HOME` or configure it in `hadoop-env.sh` and `yarn-env.sh`. In a future specification, `yarn-env.sh` will be removed.
 
 -   ODP Platforms MUST have all of the base Apache Hadoop components installed.
 
--   ODP Platforms MUST pass the Big Top smoke tests.
-> Comment (AFG) I assume this means the ODP specific Big Top tests rather than generic Apache Big Top ones.  This should be made explicit.
+-   ODP Platforms MUST pass the Apache Big Top 1.0.0 Hadoop smoke tests.
 
 -   ODP Platforms MUST NOT change public APIs, where an API is defined as either a Java API (aka "Apache Hadoop ABI") or a REST API. See the [Apache Hadoop Compatibility guidelines](http://hadoop.apache.org/docs/r2.7.1/hadoop-project-dist/hadoop-common/Compatibility.html#Java_Binary_compatibility_for_end-user_applications_i.e._Apache_Hadoop_ABI) for more information.
 
--   ODP Platforms MUST NOT modify version strings output by Hadoop components, such as those displayed in log files, or returnd via public API's.
+-   ODP Platforms MUST modify the version string output by Hadoop components, such as those displayed in log files, or returned via public API's such that they contain `-(vendor string)` where `(vendor string)` is an appropriate entry for the ODP Platform vendor in the output.
 
--   An ODP Platform MUST keep the same basic directory layout and content as the equivalent Apache component. Changes to that directory layout MUST be enabled by the component itself with the appropriate configurations for that layout configured.
-> Comment (SCG): A clarifying example here would be useful
+-   An ODP Platform MUST keep the same basic directory layout and content as the equivalent Apache component. Changes to that directory layout MUST be enabled by the component itself with the appropriate configurations for that layout configured.  For example, if Apache Hadoop YARN's package distribution contains a share directory with content, then that share directory with the equivalent content must be preset.
```

Comment:
This is a tricky one to describe. :)

Basically, Hadoop allows one to pull common, mr, yarn, and hdfs out into separate directories.  It also provides some (incomplete) ways to re-arrange the layout within that directory.  This functionality is *specifically* built for distribution makers to do different types of layouts.  In the specific case of YARN, the two (relevant to this spec) configurations are HADOOP_YARN_HOME and YARN_LIB_JARS_DIR.  

That said, this is trying to say that if HADOOP_YARN_HOME contains a share directory, then ODP Platforms should also have a share directory under their HADOOP_YARN_HOME.  The jars are *normally* stored under YARN_LIB_JARS_DIR, which by default is share/hadoop/yarn.  If the distribution wanted to move the jars to somewhere else, they WOULD be allowed to move them somewhere else (even using a different directory structure) if and only if they also change YARN_LIB_JARS_DIR to point to that new location because Hadoop allows one to move it.

All of that said, it's clear my attempt at clarifying this in this draft failed. :)  It's good feedback and I'll work on something more comprehensive.  :)